README.md

```diff
@@ -227,6 +227,25 @@ jobs:
           browser: chrome
 ```
 
+Note that if you are using Firefox and a container, you need to pass a `--user 1001` to the options for the container, as Firefox cannot be (ran as root)[https://github.com/cypress-io/cypress-docker-images/issues/85#issuecomment-849705914].
```

Comment:
My understanding is that Cypress can only access Firefox from user `1001` and the explanation that has been given it that this user matches the permissions on the `HOME` folder, however I am no expert on Docker images.

"ran as root" should be "run as root".

`(ran as root)[https://github.com/cypress-io/cypress-docker-images/issues/85#issuecomment-849705914]` is incorrect Markdown syntax. You can check how this appears by viewing https://github.com/alexjyong/github-action/tree/update-readme-firefox-container#docker-image

I suggest **not** to link to this issue, since it does not provide any extra information and in fact links back to this README as a circular reference.

Reply:
I removed the link. 

Since I'm not 100% clear why the user 1001 requirement exists, I'm just going to state it is required without an explanation as to why. If someone does have a link to a clear reason, we can add that in then.

This will be completely resolved on my end on the next commit.